components/financial_assistance/app/views/financial_assistance/incomes/_self_employed_income_form.erb

```diff
@@ -1,40 +1,87 @@
-<%= income_form_for @application, @applicant, income do |f| %>
-  <% income_id = income.id %>
-  <% onchange_date = FinancialAssistanceRegistry[:income_and_deduction_date_warning].enabled? ? (raw "validateDateWarnings('#{income_id}')") : "" %>
-  <%= f.hidden_field :kind, value: 'net_self_employment' %>
-  <div class="focus_effect_module row-edit-border">
-    <div class="row row-form-wrapper no-buffer row-height">
-      <div class="col-md-3">
-        <%= f.label :frequency_kind, 'Amount *', class: "amount label-color" %>
-        <div class="faa-amount">
-          <%= f.text_field :amount, placeholder: '0.00', value: number_to_currency(income.amount), required: true %>
+<% if @bs4 %>
+  <%= income_form_for @application, @applicant, income do |f| %>
+    <%= hidden_field_tag :bs4, true %>
+    <div class="my-4">
+      <div class="alert alert-info d-flex" role="alert">
+        <div class="pl-2 pt-2 pr-1">
+          <div class="info-icon icon" alt="info" aria-hidden="true">&nbsp;</div>
+        </div>
+        <div class="mt-1 pl-0 pr-2 pt-1">
+          <p class="subtext"><%= l10n('faa.incomes.job_end_date_note', subject: @applicant.first_name, income_type: "self-employment") %></p>
```

Comment:
Can we reuse the faa.incomes.income_note for this content? If so, we can get rid of this key in the translations file as well